spec/lib/use_case/data_export/theme_spec.rb

```diff
@@ -13,45 +14,47 @@
 
   context "when user has a theme" do
     let!(:theme) do
-      travel_to(Time.utc(2022, 12, 10, 13, 37, 42)) do
-        FactoryBot.create(:theme, user:)
+      travel_to(Time.utc(2022,
+                         12,
+                         10,
+                         13,
+                         37,
+                         42,)) do
+        FactoryBot.create(:theme,
+                          user:,)
       end
     end
 
     it "returns the theme as json" do
       expect(json_file("theme.json")).to eq(
         {
           theme: {
-            id:                 theme.id,
```

Comment:
no more ids or timestamps?

Reply:
I mean, we can add them back, just need to do the conditional logic inside the field collecting to skip over them (otherwise it hex's them too).

But to the end user they aren't really that important, I'd say.